### ISSUE TITLE
[REQUEST] Disable/ Enable boss-specific functions

### ISSUE THREAD

**Issue by Babette-Lordearon** [Zidras/DBM-Warmane#179]
### Is your feature request related to a problem? Please describe.

Some nab was using an older version of DMB and it auto-removed DI.

### Desired solution

Is there a way for a raid lead to disable (/ enable) functions of older DBM verisons, such as the "Remove DI" function on Saurfang? 

### Describe alternatives you've considered

The option to force an update of DBM automatically.

Perhaps needless to say; but I'd like to suggest to never enable any boss-specific function automatically. So that when slackers use outdated dbm-versions, they cannot mess stuff up. (Other options that won't mess up mechanics are fine though; such as unequip for Lady DW. But remove DI shouldn't be automatic, if nabs can't click it off, better let them stand there than risk something worse).

**Comment by Fanquar** [comment]
no, there isn't, as there would need to be something on the older dbm to "listen" to such input from a raidleader

also would be potentially abusable
